### PR TITLE
Disable new version check

### PR DESCRIPTION
**Pull request by LocutusOfBorg**
followup PR after https://github.com/BOINC/boinc/pull/4272

### CODE REVIEW

**Comment by LocutusOfBorg** [comment]
note: I don't know how to test because I didn't yet figure out how the parsing works (and how to change the xml file)

**Comment by talregev** [comment]
copy this xml:
https://github.com/BOINC/boinc/blob/master/android/BOINC/app/src/main/assets/cc_config.xml

add these lines:

```xml
 <disable_version_check>1</disable_version_check>
```

**Comment by LocutusOfBorg** [comment]
well so my conf file was correct...
cat /etc/boinc-client/cc_config.xml
```
<!--
This is a minimal configuration file cc_config.xml of the BOINC core client.
For a complete list of all available options and logging flags and their
meaning see: https://boinc.berkeley.edu/wiki/client_configuration
-->
<cc_config>
  <log_flags>
    <task>1</task>
    <file_xfer>1</file_xfer>
    <sched_ops>1</sched_ops>
  </log_flags>
  <options>
    <disable_version_check>1</disable_version_check>
  </options>
</cc_config>
```

but for some reasons I still see the "update" mechanism shown...

**Comment by LocutusOfBorg** [comment]
I'm debugging the parser code

**Comment by talregev** [comment]
I think you need to delete the  xml you copy.
 let boinc create one by default, the search for it, and edit with correct option.
Then you understand what to do.

File: lib/cc_config.cpp [hunk 1]
```diff
@@ -389,6 +391,7 @@ int CC_CONFIG::parse_options(XML_PARSER& xp) {
             continue;
         }
         if (xp.parse_int("max_event_log_lines", max_event_log_lines)) continue;
+        if (xp.parse_bool("disable_version_check", disable_version_check)) continue;
```
Comment (AenBleidd) [inline lib/cc_config.cpp:394]: You already parsed this flag on the line 322

**Review by AenBleidd — CHANGES_REQUESTED** [review]

**Comment by talregev** [comment]
@LocutusOfBorg Do you need some help?

**Comment by AenBleidd** [comment]
Since we have now a CI flow that builds the linux packages for 4 different Linux distros (for more details see https://github.com/BOINC/boinc/wiki/Linux-DEB-and-RPM-support), and we also want to encourage our users to use the latest version of our software, there is no reason to implement this functionality.
For the distros we're currently missing, it's quite easy to add them and publish the same way from the BOINC official web-site (and this is something we are gonna do).
Thus, I'm closing this PR without merge.

**Pull request closed without merging**